lib/compat/wordpress-6.0/block-template-utils.php

```diff
@@ -99,10 +99,18 @@ function gutenberg_generate_block_templates_export_file() {
 	// Load theme.json into the zip file.
 	$tree = WP_Theme_JSON_Resolver_Gutenberg::get_theme_data();
 	$tree->merge( WP_Theme_JSON_Resolver_Gutenberg::get_user_data() );
+	$theme_json_raw = $tree->get_data();
+	// If a version is defined, add a schema.
+	if ( $theme_json_raw['version'] && in_array( $theme_json_raw['version'], array( "1", "2" ) ) ) {
+		// Put $schema at the start of the array.
+		$theme_json_raw = array_reverse( $theme_json_raw );
+		$theme_json_raw['$schema'] = "https://json.schemastore.org/theme-v1.json";
```

Comment:
Would it be better to use this URL: https://schemas.wp.org/trunk/theme.json?

I think this one is always kept up to date with Gutenberg, based on [this post](https://make.wordpress.org/themes/2021/11/30/theme-json-schema/).